docs/development/ceps/accepted/cep-001.rst

```diff
@@ -5,12 +5,12 @@
 CEP 1 - Purpose and guidelines
 **********************************
 
-* Status: draft
-* Discussion:  NA
-* Date accepted: NA
-* Last revised: Febuary 10, 2023
+* Status: accepted
+* Discussion: `PR #2256`_
+* Date accepted: 2023-03-06
+* Last revised: 2023-03-06
 * Author: Tomas Bylund
-* Created: Febuary 10, 2023
+* Created: 2023-02-23
```

Comment:
How do you end up with these dates?
From the discussion in the PR, this seems to rather be the date where the proposal was accepted (although the merge was later)?

Reply:
Sorry, brain fart (23 was from the year, not the date). The PR was opened 2023-02-10, fixed now.